lib/formtools/plugins/plugins-helpers.js

```diff
@@ -252,6 +242,50 @@ var filtersDefault = function (filters) {
 
 };
 
+var exportDefault = function (exports) {
+
+    if (exports === undefined || exports === null) {
+        return undefined;
+    }
+
+    var validatedExports = [];
+
+    // linz will accept an object, or an array of objects
+    if (!Array.isArray(exports)) {
+        exports = [exports];
+    }
+
+    // loop through each export, and default some values
+    exports.forEach(function (exp) {
+
+        // we must have a label, otherwise there is not enough information
+        if (!exp.label) {
+            return;
+        }
+
+        if (exp.action && exp.action === 'export') {
+            throw new Error('You can not define a model export with an action property of export, this is reserved for Linz.');
+        }
+
+        var expObj = {
+                label: exp.label,
+                action: exp.action || 'export',
+                enabled: !(exp.enable === true),
```

Comment:
Just thinking of a scenario where you would use that? Since, you would define an export when you need it and when would you define an export and disabled it?